LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This should be "The MIT License", I recommend copy-pasting the license from pixi itself: https://github.com/pixijs/pixi.js/blob/dev/LICENSE